lib/lock.php

```diff
@@ -62,6 +65,7 @@ class Lock {
 	 */
 	public function __construct($path) {
 		$this->path = Filesystem::normalizePath($path, true, true);
+		$this->log = \OC::$server->getLogger();
```

Comment:
Should be injected but since this is instantiated via a wrapper wasn't sure the best way to do that or if it was possible...